frontend/hooks/useMasterBalances.ts

```diff
@@ -13,24 +13,23 @@ import { Optional, WalletBalance } from '@/types';
 import { areAddressesEqual, formatUnitsToNumber, sumBigNumbers } from '@/utils';
 
 /** Check if a balance requires funding (greater than 0) */
-const requiresFund = (balance: number | undefined) => {
+const requiresFund = (balance: string | undefined) => {
   if (isNil(balance)) return false;
-  return isFinite(balance) && balance > 0;
+  return BigInt(balance) > BigInt(0);
```

Comment:
I think we shouldn't convert to bigint what's already formatted and in eth. we should only use big ints for wei and calculation of big integers